package.json

```diff
@@ -22,6 +23,7 @@
     "publish": "npm run build && node scripts/publish",
     "storybook": "NODE_ENV=development start-storybook -p 8180 -c .storybook",
     "test": "npm run test:lint && npm run test:lint:css && npm run test:unit && npm run test:packages",
+    "test:chromatic": "chromatic test run --build-script-name build:storybook || true",
```

Comment:
The `|| true` is to prevent it from ever failing the build, right?

Reply:
Correct and I do not believe so,  `|| true` will stop chromaticqa failing the pipeline however the chromaticqa status check will still fail. and eventually that status check will be marked as required to prevent merging if these new tests do fail.